sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py

```diff
@@ -646,7 +645,7 @@ def train(args: argparse.Namespace) -> nl.Trainer:
     ]
 
     if args.lora_finetune:
-        callbacks.append(ModelTransform())
+        callbacks.append(lora_transform)
```

Comment:
could you please update https://github.com/NVIDIA/bionemo-framework/blob/67bc54f965c2579d6c0a49afeb8e6e8a957e8b0b/ci/benchmarks/partial-conv/evo2_finetuning.yaml#L63 with required changes?

it can be done in the followup mr